### PR TITLE
document descending order of resolutions array

### LINKED ISSUE #12354: If `resolutions` is set on view, `resolution` is ignored
**Describe the bug**
If a map is created with `resolutions`, the value of `resolution` is ignored. Instead `resolution` is set to the lowest value of `resolutions`. If `resolution` is not set, the map is not rendered.

**To Reproduce**
See console output of https://codesandbox.io/s/simple-forked-mhm4e?file=/main.js

**Expected behavior**
I would expect that the view would initialize with the set resolution.

### PR DESCRIPTION
**Pull request by simonseyock**
Fixes #12354

### CODE REVIEW

**Comment by mike-000** [comment]
Perhaps there should be an explanation of why, such as  "If specified the resolutions define the view zoom levels and must be in descending order."

**Comment by simonseyock** [comment]
Yeah, maybe rephrase the whole thing, because the main purpose of `resolutions` is not the constraint, but setting the zoom levels. Good point!

**Review by tschaub — APPROVED** [review]

**Comment by tschaub** [comment]
Thanks, @simonseyock.
